### DISCUSSION TITLE
New Osu Mania Touch Input Makes It Hard to Play

### DISCUSSION THREAD

**Discussion by c0rty0** [ppy/osu#28361] [Ideas] [answered]
The new osu mania touch input that was inplemented makes it hard to play, and I believe there should be a option to choose between having the new touch input and the old touch input. The old touch input was incredible as you were able to hit notes directly in the lane they were falling. Now the new touch input makes it so my fingers are spread out across the whole ipad and now it makes it difficult in hitting the notes right corresponding to their lane. The old touch input allowed you to just click directly in the lane, which was extremly efficient and helpful. The new touch input is spread across the whole screen and is weird as the touch input doesn’t line up with the lanes. The old touch input allowed me to play osu mania efficiently and happily, but now the new touch input makes me have to change how i play and I can’t play as well as I did before. Also, there are other mobile games out there that use something similar to new touch input, which is why i moved to osu lazer for mania because it was different and made sense for you to tap directly on the lane to hit the note. So, there should be an option to switch between the new and old touch input for osu mania.

**Comment by Walavouchey** [comment]
the touch areas are planned to be customisable at least (so you can put them on top of the lanes) as mentioned in https://github.com/ppy/osu/pull/28173

**Comment by jovijera** [comment]
It is indeed realy bad, i hope they revert it back to how i was.

**Reply by peppy** [reply, marked as answer]
There will be settings soon.

**Reply by IAmREGE** [reply]
In my opinion, the note track and the input area should be customizable (through skin), and that's the solution.

**Reply by oblessnoob** [reply]
> There will be settings soon.很快就会有设置。

Hi, where is the setting?[This pull](https://github.com/ppy/osu/pull/28173) forced me still using [2024.412.1](https://github.com/ppy/osu/releases/tag/2024.412.1)

**Comment by arily** [comment]
hi do we have any update on this?

**Reply by Joehuu** [reply]
Maybe https://github.com/ppy/osu/pull/31368. This change will make it possible to play in landscape.
